src/Mutation.php

```diff
@@ -89,21 +94,25 @@ public function getMutatedNodeClass(): string
 
     public function getHash(): string
     {
-        $mutatorClass = get_class($this->getMutator());
-        $attributes = $this->getAttributes();
-        $attributeValues = [
-            $mutatorClass,
-            $attributes['startLine'],
-            $attributes['endLine'],
-            $attributes['startTokenPos'],
-            $attributes['endTokenPos'],
-            $attributes['startFilePos'],
-            $attributes['endFilePos'],
-        ];
-
-        $hashKeys = array_merge([$this->getOriginalFilePath(), $mutatorClass], $attributeValues);
-
-        return md5(implode('_', $hashKeys));
+        if (!isset($this->hash)) {
```

Comment:
What do you think about comparing with `null` instead?

Reply:
I intended to do that at first. But i found isset being used elsewhere in the code.